packages/docs/src/styles/_examples.scss

```diff
@@ -65,7 +65,7 @@ $options-margin: $pt-grid-size * 3;
     margin-bottom: $options-margin;
   }
 
-  .pt-input-group + .pt-input-group {
+  :not(.pt-control-group) .pt-input-group + .pt-input-group {
```

Comment:
// Copied from the other PR (sorry for the split contexts)

The previous global examples style broke the `DateRangeInput`, because the component includes two sibling `.pt-input-group`s:

![image](https://cloud.githubusercontent.com/assets/443450/22573914/59f9053e-e960-11e6-8f4b-851724f03245.png)

Needed to tweak the global style to fix this.